src/frame/sdk/GroupApi.ts

```diff
@@ -84,18 +85,46 @@ export function get(groupID: string) {
     return GroupApiEndpoints.callGroupGetApi(groupID).map(formatDetailedGroupResponse);
 }
 
+function createErrorForInvalidUserList(userKeys: UserKeyListResponseType, userGrants: string[]) {
+    const existingUserIDs = userKeys.result.map(({id}) => id);
+    const missingUsersString = userGrants.filter((userID) => existingUserIDs.indexOf(userID) === -1);
```

Comment:
We should use `Array.includes` here as it's more appropriate than `indexOf`.
```suggestion
    const missingUsersString = userGrants.filter((userID) => existingUserIDs.includes(userID));
```
You might see a bunch of other code that uses `indexOf` but that was because `includes` doesn't work in IE. Now that we don't support IE we can use `includes` instead.